corehq/apps/accounting/tests/test_software_plan_version_filter.py

```diff
@@ -0,0 +1,51 @@
+from corehq.apps.accounting.models import (SoftwarePlan,
```

Comment:
nit, typically prefer imports indented like this
```suggestion
from corehq.apps.accounting.models import (
    SoftwarePlan,
    SoftwarePlanEdition,
    SoftwarePlanVersion,
    SoftwarePlanVisibility,
    SoftwareProductRate,
)
```